include/abb_librws/rws_poco_client.h

```diff
@@ -387,7 +398,17 @@ class POCOClient
   void extractAndStoreCookie(const std::string cookie_string);
 
   /**
-   * \brief Static constant for the websocket's buffer size.
+   * \brief Static constant for the default timeout for HTTP requests, in microseconds.
+   */
+  static const Poco::Int64 DEFAULT_TIMEOUT = 400000;
```

Comment:
Perhaps use `4e5`?

Reply:
l get conversion warnings if I use 4e5 (i.e. from double to Poco::Int64)

Reply:
Never mind then. It's probably not worth it to add a cast.

If we (and Poco) ever migrate to C++11 (we should put that on the roadmap perhaps) we could use types from `std::chrono`.